_posts/en/newsletters/2021-09-22-newsletter.md

```diff
@@ -0,0 +1,168 @@
+  [taproot][topic taproot], [BIP86][], and signet.
+
+## Preparing for taproot #14: testing on signet
+
+*A weekly [series][series preparing for taproot] about how developers
+and service providers can prepare for the upcoming activation of taproot
+at block height {{site.trb}}.*
+
+{% include specials/taproot/en/13-signet.md %}
+
+## Releases and release candidates
+
+*New releases and release candidates for popular Bitcoin infrastructure
+projects.  Please consider upgrading to new releases or helping to test
+release candidates.*
+
+- [Bitcoin Core 0.21.2rc2][bitcoin core 0.21.2] is a release candidate
+  for a maintenance version of Bitcoin Core.  It contains several bug
+  fixes and small improvements.
+
+## Notable code and documentation changes
+
+*Notable changes this week in [Bitcoin Core][bitcoin core repo],
+[C-Lightning][c-lightning repo], [Eclair][eclair repo], [LND][lnd repo],
+[Rust-Lightning][rust-lightning repo], [libsecp256k1][libsecp256k1
+repo], [Hardware Wallet Interface (HWI)][hwi repo],
+[Rust Bitcoin][rust bitcoin repo], [BTCPay Server][btcpay server repo],
+[Bitcoin Improvement Proposals (BIPs)][bips repo], and [Lightning
+BOLTs][bolts repo].*
+
+- [Eclair #1932][] implements the revised [anchor outputs][topic anchor
+  outputs] protocol specified in [BOLTs #824][], where all presigned
+  HTLC spends use zero fee, so no fees can be stolen.  See [Newsletter
+  #165][news165 bolts 842] for details.
+
+- [LND #5405][] extends the `updatechanpolicy` RPC so that it reports
+  any channels that are currently unusable due to the current policy (or
+  because of other issues, such as the channel funding transaction still
+  being unconfirmed).
+
+- [LND #5304][] enables LND to create and validate macaroons with external
+  permissions unknown to LND itself. This change enables tools such as [Lightning
+  Terminal][] to use a single macaroon to authenticate across multiple daemons that
+  all talk to the same LND.
+
+- [Rust Bitcoin #628][] adds support for Pay to Taproot's sighash
+  construction and tidies up storage of the sighash cache for legacy,
+  segwit and taproot inputs.
+
+- [Rust Bitcoin #580][] adds support for the P2P network messages
+  defined in the [BIP37][] specification for [transaction bloom
+  filtering][topic transaction bloom filtering].
+
+- [Rust Bitcoin #626][] adds functions to get the stripped size of a
+  block (a block with all its segwit data removed) and the vbyte size of
+  a transaction.
+
+- [Rust-Lightning #1034][] adds a function that can be used to retrieve
+  the complete list of channel balances, including for channels that are
+  currently in the process of closing.  This allows end-user software to
+  display a consistent user balance even when some funds are still
+  waiting for confirmation before they can be used again.
+
+<!-- FIXME: harding to update releases/RCs and add topic links -->
```

Comment:
I see topics have been added and it looks like the bitcoin core release entry is up to date. Pushing a commit to remove this FIXME.